wagtail/admin/templates/wagtailadmin/icons/draft.svg

```diff
@@ -1,3 +1,3 @@
-<svg id="icon-draft" viewBox="0 0 23 21">
-    <g fill-rule="evenodd"><path d="M18 18v1.11a.88.88 0 01-.875.89H4.875A.88.88 0 014 19.11V18h14zM12.167 1v5.047c0 .49.393.89.875.89H18V9H4V1.89c0-.455.332-.828.765-.883L4.875 1h7.292zm1.071 0c.25 0 .489.098.664.273l3.825 3.829a.935.935 0 01.273.66V6h-5V1h.238z" fill-rule="nonzero"/><text font-family="-apple-system,BlinkMacSystemFont,'Segoe UI',Helvetica,Arial,sans-serif" font-size="7" font-weight="bold" transform="translate(0 .333)"><tspan x="0" y="15.667">DRAFT</tspan></text></g>
+<svg id="icon-draft" viewBox="0 0 24.8 23">
+    <path d="M19.4 18.6v1.1c0 .5-.4.9-.9.9H6.3c-.5 0-.9-.4-.9-.9v-1.1h14zm-5.8-17v5c0 .5.4.9.9.9h5v2.1h-14V2.5c0-.5.3-.8.8-.9h7.3zm1 0c.2 0 .5.1.7.3l3.8 3.8c.2.2.3.4.3.7v.2h-5v-5h.2zM1 11.5h1.8c.4 0 .7 0 1 .1.3.1.5.2.8.5.2.2.4.5.5.8s.2.7.2 1.2c0 .4-.1.8-.2 1.1-.1.3-.3.5-.5.8-.2.2-.4.3-.7.4-.2.1-.5.1-.9.1H1v-5zm1.1.8v3.3h1.4c.2 0 .3-.1.4-.2.1-.1.2-.3.3-.5s.1-.5.1-.9 0-.7-.1-.9c-.1-.2-.2-.4-.3-.5-.1-.1-.3-.2-.5-.2H2.1zM6.1 16.5v-5h2.1c.5 0 .9 0 1.2.1.2.1.4.2.6.5.1.2.2.5.2.8 0 .4-.1.7-.3.9s-.6.4-1 .5c.2.1.4.3.5.4.1.2.3.4.6.8l.6 1H9.4l-.7-1.1c-.3-.4-.4-.6-.5-.7s-.2-.2-.3-.2c-.1 0-.3-.1-.5-.1h-.3v2.1h-1zm1-2.9h.7c.5 0 .8 0 .9-.1s.2-.1.3-.2.1-.2.1-.4 0-.3-.1-.4c-.1-.1-.2-.2-.4-.2H7.1v1.3zM15.7 16.5h-1.1l-.4-1.1h-2l-.4 1.1h-1.1l2-5h1.1l1.9 5zm-1.9-2-.7-1.9-.7 1.9h1.4zM16.2 16.5v-5h3.4v.8h-2.4v1.2h2.1v.8h-2.1v2.1h-1zM21.6 16.5v-4.2h-1.5v-.8h4v.8h-1.5v4.2h-1z"/>
```

Comment:
Love that we’re converting this to a path, but this is loong a bit too thin (and smaller). Let’s try a thicker text stroke and potentially larger icon?

Reply:
Unfortunately I had to use the `<text>` element in the end. When using a path and the svg is even slightly resized it causes it to become blurry. Even at regular size the path isn't as crisp to read as a text element, so i've updated the alignment now